dashboard/src/actions/apps.ts

```diff
@@ -126,7 +126,13 @@ export function getApp(
       }
       dispatch(selectApp(installedPackageDetail!, resourceRefs, availablePackageDetail));
     } catch (e: any) {
-      dispatch(errorApp(new FetchError(e.message)));
+      dispatch(
+        errorApp(
+          e instanceof Error
```

Comment:
I've just followed the pattern which you'd done elsewhere Rafa, but wondering when `e` would not be an instance of `Error`? Guessing you saw something else, but it still had an `e.message`, so confused?

Reply:
Don't really remember, probably due to the any in `} catch (e: any) {`. But given that it is a catch, I'm with you, can't think of a situation where `e` would not be an Error.

Reply:
Great, I'll update both this one and the one you'd added before landing.